graylog2-web-interface/src/views/components/widgets/Widget.tsx

```diff
@@ -240,9 +244,21 @@ const Widget = ({ id, editing, widget, title, position, onPositionsChange }: Pro
   const { config } = widget;
   const isFocused = focusedWidget?.id === id;
 
+  const view = useView();
```

Comment:
Here also, we could move the logic related to the warm tier into `WidgetWarTierAlert` and pass the necessary props. What do you think?